.travis.yml

```diff
@@ -11,8 +11,7 @@ before_script:
 script:
   - docker build --target test --tag my-test-image .
   - docker run my-test-image todo_app/test_tasks.py
-  - docker run my-test-image todo_app/test_client.py
-  - docker run -e MONGO_DB_URL=$MONGO_DB_URL  my-test-image todo_app/test_tasks.py
+  - docker run -e MONGO_DB_URL=$MONGO_DB_URL  my-test-image todo_app/test_client.py
   - docker build --target prod --tag $DOCKER_USER/todo-app:latest .  
```

Comment:
You've removed the end-to-end test, even though it's essentially working - it would be nice to switch that back on before merging if you can

Locally, the only issue I had was that you find the dotenv file but don't load it in
Inside docker, I think we just need to switch back on the headless/no-sandbox mode;
``` python
@pytest.fixture(scope="module")
def driver():
    opts = Options()
    opts.headless = True
    opts.add_argument('--no-sandbox')
    driver = webdriver.Chrome(ChromeDriverManager().install(), options=opts) 
    yield driver 
```